### DISCUSSION TITLE
Perfect

### DISCUSSION THREAD

**Discussion by markob94** [stav121/i3wm-themer#129] [General]
Hi,

This is so awesome. For newbies like me, perfect :) 

I hope some new themes will be added soon xD

**Comment by stav121** [comment]
Hello sir, 

I'm actually really glad that you like it and that people keep using it even though I haven't spent much time on this project lately.

I actually have a few instructions [here](https://github.com/unix121/i3wm-themer/tree/main/templates) that people can follow in case someone wants to submit a new theme, if you create something that you like, feel free to create a Pull Request with a new theme!
